fa24/sorting-project/group2/README.md

```diff
@@ -1,12 +1,46 @@
-TILE_GROUP_DIM_X = 4 and TILE_GROUP_DIM_Y = 2
-|Sorting Size  |Cycle Count  |	Core Utilization  |	Total Instr Executed  | Vcache Utilization  | DRAM Utilization |
-|--------------|-------------|--------------------|-----------------------|---------------------|------------------|
-|2^7	       |100559 	     |7.35 %	          |59088	              |1.86 %	            |0.10 %            |
-|2^8	       |189285	     |6.16 %	          |93264  	              |1.74 %	            |0.11 %            |
-|2^9	       |367241	     |5.50 %	          |161616    	          |1.67 %	            |0.11 %            |
-|2^10	       |722785	     |5.16 %	          |298320	              |1.64 %	            |0.11 %            |
-|2^11	       |1433755	     |4.98 %	          |571728	              |1.62 %	            |0.11 %            |
-|2^12	       |2855110	     |4.90 %	          |1118544	              |1.62 %	            |0.11 %            |
-|2^13	       |5698866	     |4.85 %	          |2212176	              |1.78 %	            |0.12 %            |
-|2^14	       |11478110	 |4.74 %	          |4350160	              |1.57 %	            |0.14 %            |
-|2^16	       |17326288	 |3.25 %	          |17326288	              |1.08 %	            |3.10 %            |
+### Results for Group_2 (Dec. 15)
+
+#### Speedup over previous leader Geomean: 386,241/454,584 = 0.85X compared with group_3
```

Comment:
Are you leveraging any prior version (other group's code) for this PR?

Reply:
Thank you for your comment! We are not leveraging any other group’s code; this is an improvement of our first version of the code.

Reply:
Thanks @Zhichenzzz !